dialert.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+date_default_timezone_set('America/New_York');
+
+function getDutySup($connection) {
+  $statement = $connection->query("SELECT dutysup FROM `crews` WHERE date = curdate()");
```

Comment:
Would this only be checked at the beginning/end of crews to update dialert? Since otherwise it seems like it would switch the duty sup prematurely at midnight

Reply:
DiALert will run twice per day: at or around 1800, it will check _today's_ duty sup and set the ring groups appropriately (duty sup and backup duty sup), and at or around 0600, it will reset the ring groups to include all the available duty sups. As such, this script only ever needs to provide today's sup.

Reply:
Got it. In that case, everything looks good to me!